osu.Game/Online/API/DummyAPIAccess.cs

```diff
@@ -11,21 +11,45 @@ public class DummyAPIAccess : IAPIProvider
         public Bindable<User> LocalUser { get; } = new Bindable<User>(new User
         {
             Username = @"Dummy",
-            Id = 1,
+            Id = 1001,
         });
 
         public bool IsLoggedIn => true;
 
-        public void Update()
-        {
-        }
+        public string ProvidedUsername => LocalUser.Value.Username;
+
+        public string Endpoint => "https://test.com";
+
+        public APIState State => LocalUser.Value.Id == 1 ? APIState.Offline : APIState.Online;
```

Comment:
Huh? This doesn't work together with `Login()`. Why not just store a simple bool?